layouts/community/list.html

```diff
@@ -3,6 +3,6 @@
 {{ end }}
 
 {{ define "post-hero" }}
-<a href="https://events.linuxfoundation.cn/events/kubecon-cloudnativecon-china-2018/"><img src="/images/kubecon_china_ad_2.jpg" alt="KubeConShanghai" width="49.9%" align="center"></a>
-<a href="https://events.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2018/"><img src="/images/kubecon_na_ad_2.jpg" alt="KubeConSeattle" width="49.9%" align="right"></a>
+<a href="https://events.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2018/"><img src="/images/kubecon_na_ad_2.jpg" alt="KubeConSeattle" width="49.9%" align="center"></a>
+<a href="https://events.linuxfoundation.org/events/kubecon-cloudnativecon-europe-2019/"><img src="/images/kubecon_na_ad_barcelona.jpg" alt="KubeConBarcelona" width="49.9%" align="right"></a>
```

Comment:
I can't find `kubecon_na_ad_barcelona.jpg` in https://github.com/kubernetes/website/tree/master/static/images